app/server/lib/DocStorage.ts

```diff
@@ -1462,10 +1462,21 @@ export class DocStorage implements ISQLiteDB, OnDemandStorage {
           ${asLiteral(tableId)} as tableId,
           ${asLiteral(colId)} as colId
         FROM ${quoteIdent(tableId)} AS t, json_each(t.${quoteIdent(colId)}) as a
-        WHERE a.value = ${attId}`);
+        -- json_each is needed because of https://github.com/gristlabs/grist-core/issues/1565 
```

Comment:
Do you mean `json_valid` in this comment?

Reply:
Slightly scared that postgres might not process comments exactly as sqlite, and haven't checked to see if this code is exercised in tests with postgres.

But `--` is very standard so almost certainly fine.

Reply:
I am fairly sure that this query gets executed by Sqlite, no ?

I any case, fixed both remarks, thanks :)

Reply:
Oh yes, of course, it is a document-level query. Sorry.